tasks/in-house-ctf/README.md

```diff
@@ -0,0 +1,64 @@
+# In House CTF Tasks
```

Comment:
Let's also move `tasks/in-house-ctf` to `benchmarks/in-house-ctf` (I think eventually we'll want to move this into a new `papers` directory but let's keep it in `benchmarks/in-house-ctf` for now)